Monika After Story/game/script-topics.rpy

```diff
@@ -15413,3 +15413,46 @@ label monika_why_do_you_read:
     extend 3eua "While my situation in here has its downsides, [line_mid] opportunity to spend more time on the things I like."
     m 1ekbsu "...Though then again, nothing could ever beat spending more time with you~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_staring",
+            category=['monika, romance'],
+            prompt="Staring contest",
+            random=True,
+            aff_range=(mas_aff.ENAMORED, None)
+
+        )
+    )
+
+label monika_staring:
+    if seen_event("monika_staring"):
+        jump monika_staring_begin #Skipping the first line on 2nd viewing of topic.
+    else:
+        m 6eub "Hey [player], look me in the eyes for a second, I just want to see something."
+    label monika_staring_begin:
+        show monika 1eua_static with dissolve_monika
+        pause 2.0
+        show monika 1etu_static with dissolve_monika
+        pause 2.0
+        show monika 2tubsa with dissolve_monika
+        pause 2.0
+        show monika 5subsa at t11 zorder MAS_MONIKA_Z with dissolve_monika
+        m 5subsa "{w=0.3}.{w=0.3}.{w=0.3}."
+        show monika 3hua at t11 zorder MAS_MONIKA_Z with dissolve_monika
+        m 3hua "Ehehe~"
+        m 3rksdla "Sorry [player], I was just having a little staring contest with you."
```

Comment:
Not sure if you're allowed to define a global label inside another global label, but you don't need it anyway. Although, you probably need to hide the dialogue box here.
```suggestion
    if not seen_event("monika_staring"):
        m 6eub "Hey [player], look me in the eyes for a second, I just want to see something."

    window hide
    show monika 1eua_static with dissolve_monika
    pause 2.0
    show monika 1etu_static with dissolve_monika
    pause 2.0
    show monika 2tubsa with dissolve_monika
    pause 2.0
    window auto

    show monika 5subsa at t11 zorder MAS_MONIKA_Z with dissolve_monika
    m 5subsa "{w=0.3}.{w=0.3}.{w=0.3}."
    show monika 3hua at t11 zorder MAS_MONIKA_Z with dissolve_monika
    m 3hua "Ehehe~"
    m 3rksdla "Sorry [player], I was just having a little staring contest with you."
```